### PR TITLE
style-guide: add Chinese translation

### PR DESCRIPTION
**Pull request by noarchwastaken**
a proposal in addition to #5240 to guide new contributors to
unify the formatting of Chinese pages.

Previously, I stated in #5240 that I personally format Chinese differently; for simplicity, I still think it's best to follow a widely used standard, the [Chinese Copywriting Guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines). Sorry for any confusion that might have caused...

- [x] The page (if new), does not already exist in the repo.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by marchersimon** [comment]
Should we maybe move this to `contributing-guides/style-guide.zh.md` instead of `contributing-guides.zh/style-guide.md`, so we don't have yet another directory?

**Review by sbrl — COMMENTED** [review]
Thanks so much, @noarchwastaken! I don't know the language, so are there any other recent contributors who know Chinese that we could ask to review this?

**Comment by marchersimon** [comment]
Maybe @BillLucky, @zjuyk or @sandylaw could take a look.

**Comment by bl-ue** [comment]
Also ping @starccy, @zhouLion, @wizarot, @shanoaice, @xiaolong-666 & @ChungZH 🙂

**Comment by bl-ue** [comment]
Oh, also @gyli, who has had input on this subject in the past.

File: contributing-guides/style-guide.zh.md
Comment (einverne) [inline contributing-guides/style-guide.zh.md:16]: I believe the end of the line should be `.` not the `。`

I remember there was some discussion before, but we still not reach an agreement.

File: contributing-guides/style-guide.zh.md
Comment (einverne) [inline contributing-guides/style-guide.zh.md:80]: 让它们被盲目复制粘贴时一定运行失败。

This sentence is a little bit hard to read. How about

请在示例命令中注意改写，使其不能被盲目地复制粘贴执行。

**Review by einverne — COMMENTED** [review]
Thank you @noarchwastaken for the proposal, I leave some comments for you to take a look.

File: contributing-guides/style-guide.zh.md
Comment (noarchwastaken) [reply contributing-guides/style-guide.zh.md:16]: As the description said, I _personally_ use `.`, but figured that it's simpler to just follow the guideline stated above.

Comment (BillLucky) [inline contributing-guides/style-guide.zh.md:16]: I also tend to use `.`

Comment (bl-ue) [reply contributing-guides/style-guide.zh.md:16]: The Chinese period looks weird even to me after the Latin characters.  Maybe just "always use the Latin period at the end of a sentence when the last character of the sentence is Latin"?

**Comment by noarchwastaken** [comment]
Emm... Judging from the current situation, I think we should do a poll somewhere.

If I got it right, we agreed on the use of spaces, and the only thing left is the punctuation; so the poll is pretty simple:

> When you are mixing English and Chinese, how do you handle punctuations?

- :heart: Always use full-width
    `“tldr”，意即“太长不读”，简直是命令行菜鸡的福音！`

- :rocket: Always use half-width
    `"tldr", 意即"太长不读", 简直是命令行菜鸡的福音!`

- :eyes: Following the text before and the quoted text
    `"tldr", 意即“太长不读”，简直是命令行菜鸡的福音！`

We may just do it here and on #5240 by reacting with the above emojis.

**Comment by sbrl** [comment]
In the English pages @noarchwastaken we explicitly use ascii `"` as opposed to "smart quotes" that you'd seen in say Microsoft Word.

**Comment by noarchwastaken** [comment]
> Best to use standard ASCII characters for legacy support anyway

If someone is using legacy terminals like the non-Unicode xterm, then it won't display Chinese altogether...

**Comment by bl-ue** [comment]
Pinging the following users: @starccy @zhouLion @wizarot @shanoaice @xiaolong-666 @ChungZH @gyli @BillLucky @zjuyk @sandylaw @onesuper @gonejack @zhengqm @zhoufeng1989

Will you please read https://github.com/tldr-pages/tldr/pull/5952#issuecomment-846608320 and react to that comment with the appropriate emoji per your opinion, as @noarchwastaken defined?  Thank you!

Let's wait a week for votes, then we can move on.  Then, I'll update and merge #5240 🎉

**Comment by gonejack** [comment]
I personally don't read Chinese translation of tldr, but the explanation of tldr

"tldr", 意即“太长不读”，简直是命令行菜鸡的福音！ 

is a bit playful, hoping this sentence not being merged already.

**Comment by noarchwastaken** [comment]
> I personally don't read Chinese translation of tldr, but the explanation of tldr
> 
> "tldr", 意即“太长不读”，简直是命令行菜鸡的福音！
> 
> is a bit playful, hoping this sentence not being merged already.

It's just a random example that I came up for the poll :thinking:

**Comment by noarchwastaken** [comment]
@bl-ue Checking in here about a week later:

The votes look a bit confusing, but my count on the votes is:
```
Full-width supporters:
@gyli @bl-ue @marchersimon @noarchwastaken

No half-width supporter

Smart quotes supporters:
@fangyi-zhou @BillLucky @gonejack 
```

So full-width it is, winning by merely one vote...

I think it's good enough, better than being stuck here for almost half a year. We can now start pushing Chinese translation forward again :tada:

**Comment by bl-ue** [comment]
Well two of those votes are not even Chinese speakers, but whatever 😆

**Comment by sbrl** [comment]
Would be more helpful if people could explain their reasoning behind their vote rather than just voting. Then we can reach a decision everyone is happy with.

Traditionally though in other areas we have gone with ASCII characters (e.g. `"` over smart quotes, `...` over `…`).

**Comment by noarchwastaken** [comment]
> Would be more helpful if people could explain their reasoning behind their vote rather than just voting. Then we can reach a decision everyone is happy with.

I think many of us already did so in #5240. I intend this poll to be a summary since #5240 is filled with discussions and hard to follow...

**Comment by gonejack** [comment]
The basic esthetic rule in Chinese typography is that every characters are equal width thus Chinese has is own punctuation marks. It would be regarded as typos in Chinese if one uses "" instead of “” around Chinese characters or uses “” around Latin alphabet.

**Comment by sbrl** [comment]
Ah, that's very interesting @gonejack. In that case, let's use the smart quotes `“”` around Chinese characters.

**Comment by marchersimon** [comment]
To me, that sounds like the most elegant solution. (Even thought I know absolutely nothing about Chinese).

File: contributing-guides/style-guide.zh.md
Comment (marchersimon) [inline contributing-guides/style-guide.zh.md:16]:
```suggestion
> 更多信息：<https://example.com>.
```

File: contributing-guides/style-guide.zh.md
Comment (marchersimon) [inline contributing-guides/style-guide.zh.md:90]:
```suggestion
1. 路径，例如 `package.json`，`/etc/package.json`.
2. 扩展名，例如 `.dll`.
3. 命令，例如 `ls`.
```

**Review by marchersimon — COMMENTED** [review]
Latin punctuation after latin symbols is what we decided upon right? I think after we apply those two, we are ready to merge

**Comment by marchersimon** [comment]
@noarchwastaken do you want to finish this PR or would you want us to take over?
Also, it this page just a mere translation of the English one, or does it contain anything about Chinese punctuation?

**Comment by noarchwastaken** [comment]
> @noarchwastaken do you want to finish this PR or would you want us to 
take over?

I think we agreed on all full-width: 
https://github.com/tldr-pages/tldr/pull/5952#issuecomment-863974990

 > Also, it this page just a mere translation of the English one, or 
doesit contain anything about Chinese punctuation?


It does. I referenced the copywriting guideline and added a summary of 
it to hopefully help with the current situation...

**Comment by marchersimon** [comment]
The decision was a bit messy and without the two non-Chinese voters, which just voted for what seemed best in the moment, it would have been the first option, which we then decided upon [here](https://github.com/tldr-pages/tldr/pull/5952#issuecomment-868796784).

> It does. I referenced the copywriting guideline and added a summary of
it to hopefully help with the current situation...

It would be really helpful if we'd had that page in English and Chinese. Would you like to translate it or should I write a new one in English, which then someone translates into Chinese?

**Comment by blueskyson** [comment]
@marchersimon I can write an English version after this one being merged.

**Comment by marchersimon** [comment]
Doesn't sound like a bad idea, but let's see what the others think. 

The problem with merging this page first is, that we cannot review it and have to wait for the English one. If some Chinese contributors would approve here I'd be fine with it tough. (@blueskyson @BillLucky if you'd like to take a look)

File: contributing-guides/style-guide.zh.md
Comment (blueskyson) [inline contributing-guides/style-guide.zh.md:41]: Here is a typo. The counterpart in style-guide is:
```suggestion
tldr --render {{page.md}}
```

File: contributing-guides/style-guide.zh.md
Comment (blueskyson) [inline contributing-guides/style-guide.zh.md:42]:
```suggestion
```
tldr --render {{page.md}}
```
```

**Comment by blueskyson** [comment]
@marchersimon Just fully read this one, I think it's clearly enough. And I would approve this after all suggestions handled.

File: contributing-guides/style-guide.zh.md
Comment (marchersimon) [reply contributing-guides/style-guide.zh.md:42]: Yep, that's not correct. Above it says `` `-f` (`--render``) ``, which should be `-r`

**Comment by ChungZH** [comment]
I think this PR should be completed as soon as possible. It's important

**Comment by marchersimon** [comment]
We have to finish #6285 first, since we can't review a Chinese-only document.

File: contributing-guides/style-guide.zh.md
Comment (blueskyson) [reply contributing-guides/style-guide.zh.md:90]:
```suggestion
1. 路径，例如 `package.json`，`/etc/package.json`.
2. 扩展名，例如 `.dll`.
3. 命令，例如 `ls`.

## 中西文混排规则

中文、西文、阿拉伯数字写在同一个句子时，需要注意排版。

以下规则适用于中文（zh）和繁体中文（zh_TW）：

1. 在西文单词和数字前 / 后放置一个空格。
   例如：`列出所有 docker 容器`而不是`列出所有docker容器`。 
   例如：`宽度为 50 个字`而不是`宽度为50个字`。
2. 除了度数和百分比，在数字和单位之间留一个空格。
   例如：`容量 50 MB`而不是`容量 50MB`。
   对于度数和百分比：使用`50°C`和`50%`而不是`50 °C`和`50 %`
3. 不要在全角标点符号前 / 后放置空格。
   例如：`开启 shell，进入交互模式`而不是`开启 shell ，进入交互模式`。
4. 除了西文長句，一律使用全角标点符号。
   例如：`嗨，你好。`而不是`嗨, 你好.`。
5. 当最句子最后一个字符是半角时，使用半角标点符号来结束句子。
   例如：`将代码转化为 Python 3.`而不是`将代码转化为 Python 3。`。
6. 使用精准的专有名词，不要使用非官方的中文缩写。
   例如：`Facebook`而非`facebook`、`fb`或`脸书`。
   
为保持可读性和一致性，将页面翻译成中文时，请尽可能遵守以上 6 条规则。

有关更多中西文混排规则的资讯，请参考[《中文文案排版指北》](https://github.com/sparanoid/chinese-copywriting-guidelines)。

```

**Review by blueskyson — CHANGES_REQUESTED** [review]
Can I edit the file directly as the author haven't been active for long time?

**Comment by marchersimon** [comment]
> Can I edit the file directly as the author hasn't been active for long time?

I'd say so. Go ahead.

**Comment by blueskyson** [comment]
@zjuyk @BillLucky Could you review this document? As I'm not a native simplified Chinese speaker, there might exist idioms that are rarely used.

File: contributing-guides/style-guide.zh.md
Comment (zjuyk) [inline contributing-guides/style-guide.zh.md:86]: better to remove '/'

File: contributing-guides/style-guide.zh.md
Comment (zjuyk) [inline contributing-guides/style-guide.zh.md:92]: The same. Chinese user can understand it without '/'. maybe better to remove it.

File: contributing-guides/style-guide.zh.md
Comment (zjuyk) [inline contributing-guides/style-guide.zh.md:91]: add a space before and after quote characters.

对于度数和百分比：使用`50°C`和`50%`而不是`50 °C`和`50 %`
to
对于度数和百分比：使用 `50°C` 和 `50%` 而不是 `50 °C` 和 `50 %`

**Review by zjuyk — COMMENTED** [review]
May be we should add a rule "we should always add a space before and after back quote characters."

File: contributing-guides/style-guide.zh.md
Comment (blueskyson) [reply contributing-guides/style-guide.zh.md]: My fault, the quoted characters are half-width so the spaces are needed.

Comment (zjuyk) [reply contributing-guides/style-guide.zh.md]: I noticed that you use '。' after `50 %`. This may be against with rule 5?

Comment (blueskyson) [reply contributing-guides/style-guide.zh.md]: Oh yes, I violate rule 5. This is actually an unfriendly rule.

**Comment by zjuyk** [comment]
LGTM. waiting for others suggestions

**Comment by blueskyson** [comment]
> May be we should add a rule "we should always add a space before and after back quote characters."

I feel it doesn't matter if a back quote located between Chinese characters without spaces. Thank you for reviewing. @zjuyk

**Review by blueskyson — APPROVED** [review]
I think the document is ready for merge. @marchersimon

**Review by marchersimon — APPROVED** [review]
Great, thanks!

File: contributing-guides/style-guide.zh.md
Comment (BillLucky) [inline contributing-guides/style-guide.zh.md:94]: Maybe the traditional Chinese character "長" can be replaced with the simplified Chinese character "长"。

Comment (ChungZH) [reply contributing-guides/style-guide.zh.md:94]: https://github.com/tldr-pages/tldr/pull/6391

**Pull request merged**
